src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/controllers/PostTaskConfigurationTest.java

```diff
@@ -36,7 +36,7 @@ public void cleanUp() {
 
     @Test
     public void should_return_task_configuration_then_expect_task_is_auto_assigned() throws Exception {
-        caseId = createCcdCase();
+        caseId = createCcdCase("case_data.json");
```

Comment:
Looks good minor suggestion to have an alternative to having to pass in the name of the json file, we could wrap this into:

createCcdCase()
createCcdCaseWithCaseManagementCategory()

What do you think?

Reply:
that is a good idea